templates/hash.html

```diff
@@ -1,7 +1,7 @@
 {{! This Source Code Form is subject to the terms of the Mozilla Public
     License, v. 2.0. If a copy of the MPL was not distributed with this
     file, You can obtain one at http://mozilla.org/MPL/2.0/. }}
-<!DOCTYPE HTML>
+<!doctype html>
```

Comment:
It's mostly nitpicking :)

https://github.com/h5bp/html5-boilerplate/issues/1522